taipy/core/submission/submission.py

```diff
@@ -132,6 +132,21 @@ def jobs(self) -> List[Job]:
 
         return jobs
 
+    @property
+    def running_jobs(self) -> _SelfSetterSet:
+        self._running_jobs = _Reloader()._reload(self._MANAGER_NAME, self)._running_jobs
```

Comment:
💭 So many reloads...
1. I thought there were decorators to avoid such boiler plate code? (ex: ` @_self_setter(_MANAGER_NAME))`)
2. I am a bit concerned by performance...

Reply:
you're right! Hmmm, what if I save everything `running_jobs`, etc. in a properties like `properties`. We can retrieve the dictionary before checking?

Reply:
One more thing: this public method exposes an internal type. We should not expose the `_SelfSetterSet` IMO.
How about `def running_jobs(self) -> Iterable[Job]`?

Reply:
make sense to me :D